models/tasks.js

```diff
@@ -14,6 +14,9 @@ const { TASK_TYPE, TASK_STATUS } = require("../constants/tasks");
 const updateTask = async (taskData, taskId = null) => {
   try {
     taskData = await toFirestoreData(taskData);
+    if (taskData.userNotFound) {
```

Comment:
Will we be able to update tasks if there is no **assignee**?

Reply:
Yes absolutely, there is a check in `toFireStoreData` function
https://github.com/Real-Dev-Squad/website-backend/blob/e4bb6ca1e299d15baf8d91db245e82335b077acf/utils/tasks.js#L37